azure-pipelines-json-verify.yml

```diff
@@ -0,0 +1,24 @@
+pr:
+  branches:
+    include:
+    - main
+    - dev
```

Comment:
Nit: i don't think dev and v2.x are active branches. We only use main branch for everything

Reply:
Sounds good; I'll remove those.